_includes/templates/summaries/linkedin.html

```diff
@@ -0,0 +1,16 @@
+<h2 class="h4">
+    <span class="cf-icon cf-icon-linkedin-square"></span>
+    Follow us on LinkedIn
+</h2>
+<p class="short-desc">
+    The CFPB is one of the most searched for agencies
+    in the federal government.
+    Connect with us to stay updated on the work
+    we do and new opportunities to be a part of it.
+</p>
+<p class="short-desc">
```

Comment:
There's `.block-link`, but it's mainly for adding padding and borders at small screen sizes.

I don't see a problem with a block having an inline-block sibling, if it renders fine. If you were to just apply `display: block` to it, the bottom border would stretch to the full width of the column.